include/signal.h

```diff
@@ -0,0 +1,50 @@
+#ifndef _SYS_SIGNAL_H_
+#define _SYS_SIGNAL_H_
+
+#include <common.h>
+#include <bitstring.h>
+#include <queue.h>
+#include <mutex.h>
+#include <signum.h>
+
+typedef struct proc proc_t;
+
+typedef bitstr_t sigset_t[bitstr_size(SIG_LAST)];
+
+typedef void (*sighandler_t)(int);
+
+#define SIG_DFL (sighandler_t *)0x00
+#define SIG_IGN (sighandler_t *)0x01
+#define SIG_TERM (sighandler_t *)0x02
```

Comment:
*W.r.t. header files... I'll happily accept any other solution that will reduce likelihood of user and kernel space going out of sync with common (constants, types, etc.) declarations. AFAIU unix solution to the problem is to store kernel header files in `/usr/include/sys` and include them from corresponding files in `/usr/include/`.*